acos_client/v21/action.py

```diff
@@ -39,5 +39,9 @@ def reload(self, write_memory=False, **kwargs):
     def activate_and_write(self, partition, **kwargs):
         write_format = "active-partition {0}\r\nwrite memory\r\n"
         post_body = write_format.format(partition)
-
-        return self._request("POST", "cli.deploy", params=None, payload=post_body, **kwargs)
+        # Request raises an exception when the "maybe error" is returned.
+        # We can't do anything about "maybe" errors or otherwise at this stage
+        try:
+            return self._request("POST", "cli.deploy", params=None, payload=post_body, **kwargs)
+        except:
```

Comment:
Yeah, I had thought about that and then had the thought of  "we can't do anything about it if it does fail except notify the user".   Matching on string and re-raising if it's not the droid we're looking for seems reasonable - then we can have any error issues handled at the consumer level.